packages/commonwealth/server/routes/status.ts

```diff
@@ -379,9 +375,6 @@ async function computeKnockJwtToken(userId: number) {
       signingKey: config.NOTIFICATIONS.KNOCK_SIGNING_KEY,
       expiresInSeconds: config.AUTH.SESSION_EXPIRY_MILLIS / 1000,
     });
-  } else {
-    log.warn('No process.env.KNOCK_SIGNING_KEY defined');
-    return '';
```

Comment:
`KNOCK_SIGNING_KEY` is still marked as optional in `/libs/adapters/src/config.ts`, not sure If we should remove this warning?

Reply:
@burtonator

Reply:
Yes.  This is removed.  Are you looking at a stale diff?

Reply:
I mean shouldn't we keep it as `KNOCK_SIGNING_KEY`  is optional

Reply:
It's required if FLAG_KNOCK_INTEGRATION_ENABLED is on but I think I removed it in the first place because either you or Marcin felt it was too aggressive :) I think it was Marcin though.